src/pages/docs/creators/ray/quickstart.md

```diff
@@ -0,0 +1,100 @@
+---
+description: quick and easy, copy and run instructions on launching and decommissioning the Ray on Golem cluster
+title: Ray on Golem QuickStart
+type: guide 
+---
+
+# Quickstart
+
+This quickstart shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
+It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](/docs/creators/ray/setup-tutorial)
+
+
+## Install software
+
+The first step is installing Ray and Ray on Golem (recommended within a clean virtual environment)
+
+```bash
+# install ray & ray-on-golem
+pip install -U ray[default] ray-on-golem
+```
+
+For now, you also need to download and install Golem node software representing you in the Golem network.
+
+```bash
+# install yagna - golem network daemon
+curl -sSf https://join.golem.network/as-requestor | bash -
+```
+
+## Start `ray-on-golem` server
+
+For the time being you need to manually run `ray-on-golem` server (in a separate terminal)
+
+```bash
+ray-on-golem
+```
+
+## Set the cluster up
+
+With the packages in place, you can download our sample golem cluster configuration yaml, and use it with `ray up` to start up the cluster.
+It will give you a cluster of one node (which will expand when you feed it with work) on the Golem test network (free, but not very powerful)
+
+
+```bash
+# Download the golem-cluster.yaml
+wget https://github.com/golemfactory/golem-ray/blob/main/golem-cluster.yaml
+
+# In this command:
+# * yagna starts in the background (if not running)
+# * ray-on-golem cluster manager starts in the background
+# * ray head node is started on a golem provider
+ray up golem-cluster.yaml
+
+```
+
+## Execute a Ray application
+
+Download our example Ray app and execute it locally (a Ray instance will be created on your machine)
+
+```bash
+# Download the example Ray app
+wget https://github.com/golemfactory/golem-ray/blob/main/examples/simple-task.py
+
+# Execute the app locally by starting a local ray instance on your computer
+python simple-task.py
+```
+
+Execute the app on the cluster. Observe how Ray on Golem cluster expands during the computation
+
+```bash
+# Run some ray-based code (that knows *nothing** about Golem) - this will either:
+# A) Run only on one node (the head node), if the autoscaler decides there is no need for a worker node
+# B) Or create worker node(s) on the Golem Network. Worker nodes will be later auto-terminated by the autoscaler)
+
+# Submit the app to be executed on your cluster
+ray submit golem-cluster.yaml simple-task.py 
```

Comment:
This will output dashboard ip which wont work, not sure if we can do anything about it

Reply:
you mean that it would be a good place to add info about dashboard?
What I meant by observe is that the simple-task itself prints information about cluster before it starts and after it ends...